### ISSUE TITLE
Could you provide the grid search result? 

### ISSUE THREAD

**Issue by SimmerChan** [pfnet-research/contextual_augmentation#1]
I wanna reproduce your experiment results. Could you provide the parameter searching space and the best combination?
I mean:
learning rate searching space: [0.010, 0.003, 0.01 ......]
dropout searching space: [0.1, 0.2, 0.3 ......]

Thanks!

**Comment by dendisuhubdy** [comment]
@SimmerChan rather than grid search you can use random search with Oríon (https://github.com/mila-udem/orion).

**Comment by soskek** [comment]
Typical spaces can be used. Not so sensitive, though.
e.g.
"learning_rate": [1e-3, 1e-4],
"unit": [256, 512],
"dropout": [0.1, 0.5].
